reference-implementation/to-upstream-wpts/writable-streams/write.js

```diff
@@ -197,3 +197,15 @@ promise_test(() => {
           assert_equals(writeCount, numberOfWrites, `should have called sink's write ${numberOfWrites} times`));
   });
 }, 'a large queue of writes should be processed completely');
+
+promise_test(() => {
+  const stream = recordingWritableStream();
+  const w = stream.getWriter();
+  const WritableStreamDefaultWriter = w.constructor;
+  w.releaseLock();
+  const writer = new WritableStreamDefaultWriter(stream);
```

Comment:
Nah, this is intended, just a little silly. IMO constructors should only be prohibited from working when doing so would break an invariant (like 1 controller <-> 1 stream). That's not the case here so it's good to have a working constructor, even if it's hidden by default.